src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryMath.cs

```diff
@@ -689,17 +689,17 @@ internal static double Round(double number, double digits, RoundType rt = RoundT
                 return number;
             }
 
-            var m = Math.Pow(10d, -dg);
-            var eps = m / 1e12d; // used to manage rounding of 1.4499999999999999999996
+            var m = Math.Pow(10d, dg);
+            var eps = 1 / (m * 1e12d); // used to manage rounding of 1.4499999999999999999996
```

Comment:
This is great. It's a subtle fix - can you add a comment to explain why it works? (that also acts to warn future developers from accidentally undoing it)

Reply:
will do!